### PR TITLE
Translation corrected

### PR DESCRIPTION
**Pull request by KevinArtus**
[OO-MISC] Corrected translation.

https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/104
open-orchestra/open-orchestra-cms-bundle#1600

### CODE REVIEW

**Comment by alavieille** [comment]
update OO-MISC by a message more explicit for integrator

File: BackofficeBundle/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -372,7 +372,7 @@ open_orchestra_backoffice:
             content_type:
                 list: Type de contenu
             status:
-                list: Etapes de workflow
+                list: Etape de workflow
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/translations/messages.fr.yml]: This is a global problem: take the translation above and below: ``Type de contenu`` and ``UtilisateurS``

Maybe you should create a ticket to choose one time for all

**Comment by KevinArtus** [comment]
This is a global problem. Several translations are plural, others are singular. Maybe we should refacto all the translations to singular or plural to be coherent.

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**